aries-site/src/pages/templates/navigation.mdx

```diff
@@ -6,7 +6,6 @@ import { getPageDetails, getRelatedContent } from '../../utils';
 export const navPatterns = [
   'Drill Down Navigation',
   'Side to Side Navigation',
```

Comment:
I can't comment down below but in the copy we say "There are 4 different types of navigational directions..." -- ssince we removed "Reverse", we need to change this to "There are 3 different types..."

Reply:
thanks done